packages/matplotlib/meta.yaml

```diff
@@ -31,6 +37,7 @@ build:
     rm -rf $SITEPACKAGES/matplotlib/backends/web_backend
     rm -rf $SITEPACKAGES/sphinxext
     cp $PKGDIR/src/fontlist.json $SITEPACKAGES/matplotlib
+    cp $SITEPACKAGES/matplotlib/mpl-data/fonts/ttf/* $PKGDIR/../../build/fonts/
```

Comment:
`$PKGDIR/../../build/fonts/`

Isn't there an environment variable like `$BUILDDIR` that would be more appropriate here? @rth?

Reply:
Yeah, but this isn't available to us according to https://pyodide.org/en/stable/development/new-packages.html#build-post

Reply:
In that case I guess this is fine for now even though not ideal.

Reply:
Maybe we can make it available in a separate PR?

Reply:
Yeah, @rth will have to weigh in on this. In any case I don't think this PR needs to be held up on this point.